src/main/java/net/sf/jabref/sql/DatabaseUtil.java

```diff
@@ -9,9 +9,9 @@
 
 import net.sf.jabref.MetaData;
 
-public class DBImporterExporter {
+public class DatabaseUtil {
```

Comment:
Yes, good point. Previously, these methods I changed to static ones have been in the root class of a three layered inheritence hierarchy. I just moved them out. I think this is subject to another PR someday by someone. 

And I do not have a good idea what to do with them at the moment :)